recipes/pcl/all/conanfile.py

```diff
@@ -0,0 +1,570 @@
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/PointCloudLibrary/pcl"
+    topics = ("computer vision", "point cloud", "pointcloud", "3d", "pcd", "ply", "stl", "ifs", "vtk")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        # Enable/disable individual components
+        "2d": [True, False],
+        "features": [True, False],
+        "filters": [True, False],
+        "geometry": [True, False],
+        "io": [True, False],
+        "kdtree": [True, False],
+        "keypoints": [True, False],
+        "ml": [True, False],
+        "octree": [True, False],
+        "outofcore": [True, False],
+        "people": [True, False],
+        "recognition": [True, False],
+        "registration": [True, False],
+        "sample_consensus": [True, False],
+        "search": [True, False],
+        "segmentation": [True, False],
+        "simulation": [True, False],
+        "stereo": [True, False],
+        "surface": [True, False],
+        "tracking": [True, False],
+        "visualization": [True, False],
+        "cuda_common": [True, False],
+        "cuda_features": [True, False],
+        "cuda_io": [True, False],
+        "cuda_sample_consensus": [True, False],
+        "cuda_segmentation": [True, False],
+        "gpu_containers": [True, False],
+        "gpu_features": [True, False],
+        "gpu_kinfu": [True, False],
+        "gpu_kinfu_large_scale": [True, False],
+        "gpu_octree": [True, False],
+        "gpu_people": [True, False],
+        "gpu_segmentation": [True, False],
+        "gpu_surface": [True, False],
+        "gpu_tracking": [True, False],
+        "gpu_utils": [True, False],
+        "apps": [True, False],
+        "tools": [True, False],
+        # Optional external dependencies.
+        # Only used if the corresponding component is enabled.
+        "with_cuda": [True, False],
+        "with_flann": [True, False],
+        "with_libusb": [True, False],
+        "with_opencv": [True, False],
+        "with_opengl": [True, False],
+        "with_openmp": [True, False],
+        "with_pcap": [True, False],
+        "with_png": [True, False],
+        "with_qhull": [True, False],
+        "with_qt": [True, False],
+        "with_vtk": [True, False],
+        # TODO:
+        # "with_davidsdk": [True, False],
+        # "with_dssdk": [True, False],
+        # "with_ensenso": [True, False],
+        # "with_fzapi": [True, False],
+        # "with_metslib": [True, False],
+        # "with_openni": [True, False],
+        # "with_openni2": [True, False],
+        # "with_rssdk": [True, False],
+        # "with_rssdk2": [True, False],
+        # "with_qvtk": [True, False],
+        # If to enable https://github.com/PointCloudLibrary/pcl/pull/5764 
```

Comment:
```suggestion
```
The comment is a little misleading. The option itself doesn't have a lot in common with the mentioned PR. PR reduces compilation complexity when `PCL_ONLY_CORE_POINT_TYPES` is `False`. So the default value `True` doesn't enables this patch. It is good to have it though for those who will set this option to `False` so I wouldn't delete the `0001-ReportFixAboutMemoryConsumptionDuringBuild.patch`. But in my opinion there should be no comment there at all. The comment can also be changed to the documentation of the option [there](https://github.com/PointCloudLibrary/pcl/blob/3ed96c246e5c873713ec670b895469d09149a552/cmake/pcl_options.cmake#L49)

Reply:
Is it ok with my last commit?

Reply:
I think the build will fail now. Setting `PCL_ONLY_CORE_POINT_TYPES` to `True` was the whole idea to get C3I building pcl. I was talking only about removing the comment. I apologize if I confused you with how I phrased things.